src/RageSoundReader_MP3.cpp

```diff
@@ -9,14 +9,7 @@
 #include <cerrno>
 #include <map>
 
-#if defined(_WINDOWS) || defined(MACOSX)
-#include "../extern/mad-0.15.1b/mad.h"
-#ifdef _MSC_VER
-#pragma comment(lib, "libmad.lib")
-#endif //_MSC_VER
-#else
-#include <mad.h>
-#endif // _WINDOWS
+#include "mad.h"
```

Comment:
Technically the only portion not really related to all of this, but an annoyance nonetheless. Only system provided headers should use `<brackets>`. Mad is not system provided, even if installed via package manager.